app/src/main/java/waiter/Router.java

```diff
@@ -1,23 +1,31 @@
 package waiter;
 
+import java.util.AbstractMap;
 import java.util.Map;
 
 public class Router {
 
     private final Map<String, Map<String, Map<String, String>>> routes;
     private final String noMethod = "No Method";
     private final String noUrl = "No Url";
+    Map.Entry<String, Map<String, String>> simpleGet = new AbstractMap.SimpleEntry<>("/simple_get", setResponseFields("200 OK", "Content-Length:", ""));
+    Map.Entry<String, Map<String, String>> noUrlFound = new AbstractMap.SimpleEntry<>(noUrl, setResponseFields("500 Internal Server Error", "Content-Length:", ""));
 
     public Router() {
         this.routes = Map.ofEntries(
```

Comment:
If I want to add a new route, do I have to modify this `Router` class?

Reply:
As of right now yes, but Im looking into how to make it more modular with the use of the Builder Pattern, I was thinking this would be a refactor PR on its own, to make this just getting it working and then having a separate refactor, but I could try to implement it in this PR too

Reply:
👍 I think a separate refactoring ticket makes sense. I'd also throw in inspirations from other web frameworks on how to handle adding new routes/resources for example [Sinatra](http://sinatrarb.com/intro.html).